### PR TITLE
[10.0] partner_sector: allow to use in individuals

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Forward port of https://github.com/OCA/partner-contact/pull/615.

### CODE REVIEW

**Comment by MiquelRForgeFlow** [comment]
@lreficent

File: partner_sector/models/base_config_settings.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
+
+from openerp import fields, models
```
Comment (nikul-serpentcs) [inline partner_sector/models/base_config_settings.py]: Replace `openerp` to `odoo`

File: partner_sector/views/base_config_settings.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
```
Comment (nikul-serpentcs) [inline partner_sector/views/base_config_settings.xml]: @mreficent IMHO Direct use` odoo` tag

File: partner_sector/security/partner_sector_security.xml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    
+    <data noupdate="0">
```
Comment (LoisRForgeFlow) [inline partner_sector/security/partner_sector_security.xml]: odoo and remove noupdate?

Comment (MiquelRForgeFlow) [reply partner_sector/security/partner_sector_security.xml]: done

**Comment by MiquelRForgeFlow** [comment]
:green_apple:

**Review by LoisRForgeFlow — APPROVED** [review]
Code review :+1: (runbot never working in this PR :disappointed: )

**Comment by moylop260** [comment]
Now, I see runbot green
Do you have issues here yet?

**Comment by LoisRForgeFlow** [comment]
Yes, it works now, thanks!

Just FYI, over the last weeks sometimes it was imposible to access to (even green) runbot builds. Sometimes the blue button to access the log in screen was not visible (even after rebuilding the runbot) and some other times a "502 Bad Gateway" error appeared.

**Comment by LoisRForgeFlow** [comment]
@nikul-serpentcs could you please update your review?

**Review by JordiBForgeFlow — APPROVED** [review]

**Comment by JordiBForgeFlow** [comment]
Fast merge after 2 reviews.

**Comment by moylop260** [comment]
I got it
Thank you
Maybe it was the following issue:
 - https://github.com/OCA/runbot-addons/issues/176

**Pull request merged**